validator/delphi_validator/report.py

```diff
@@ -1,6 +1,9 @@
 """Validation output reports."""
 import sys
 from typing import Set, Tuple
+from delphi_utils.logger import get_structured_logger
+
+logger = get_structured_logger()
```

Comment:
I recommend passing in __name__ here as well so the logger will have the name of this calling class associated.

get_structured_logger(__name__)

(ultimately we will also likely want a filename passed in to persist to disk, e.g., return get_structured_logger(__name__, filename = params.get("log_filename"))

Reply:
done, I think.